Makefile

```diff
@@ -138,20 +138,27 @@ build-ui: clean-ui ## build the Argo CD UI extension creating the ui/extension.t
 	yarn --cwd ${UI_DIR} install
 	yarn --cwd ${UI_DIR} build
 
-.PHONY: update-openapi-ui
-update-openapi-ui:  build goreman  ## Update the OpenAPI spec from the local server.
+.PHONY: openapi-ui
+openapi-ui:  build-go goreman  ## Update the OpenAPI spec from the local server.
+	@echo "Updating OpenAPI specification..."
 	@echo "Starting the backend server"
 	goreman start backend &
 	sleep 5
 	@echo "Downloading OpenAPI spec from dev server"
 	yarn --cwd ${UI_DIR} api:download
 	killall goreman || true
 
+
 .PHONY: codegen-ui
 codegen-ui: ## Generate the UI API files based on the OpenAPI spec.
+	@echo "Generating API files..."
+	if [ -f ${UI_DIR}/src/gen/schema.yaml ]; then echo "run make openapi-ui"; fi
```

Comment:
why not make the `openapi-ui` target as a dependency of this one so it always gets executed and no need to have this `if` condition.

Reply:
Done